src/core/Expander/ExpanderTitle/ExpanderTitle.tsx

```diff
@@ -26,10 +26,8 @@ export interface ExpanderTitleProps extends Omit<HtmlDivProps, 'className'> {
   className?: string;
   /** Title for Expander */
   children?: ReactNode;
-  /** Screen reader action label for collapsed expander toggle button. E.g. "Additional information". */
-  ariaOpenText: string;
-  /** Screen reader action label for expanded expander toggle button. Should usually be the same as `ariaOpenText` */
-  ariaCloseText: string;
+  /** Screen reader action label for expander toggle button. E.g. "Additional information". */
+  ariaToggleButtonText: string;
```

Comment:
Since the next prop is `toggleButtonAriaDescribedBy`, should this be `toggleButtonAriaLabel`? In addition, the relationship between these two could be emphasized in the descriptions e.g. "will be read along with `the other prop`".